logic/teamLogic.js

```diff
@@ -0,0 +1,37 @@
+const MongoClient = require('mongodb').MongoClient
+const MsLogger = require('@first-lego-league/ms-logger').Logger()
+
+const MONGU_URI = process.env.MONGO
+
+module.exports.deleteValidation = function (params) {
+  try {
+    deleteMatchesForTeam(parseInt(params.id))
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
+function deleteMatchesForTeam (teamNumber) {
+  MongoClient.connect(MONGU_URI).then(connection => {
+    connection.db().collection('matches').find({ 'matchTeams.teamNumber': teamNumber }).toArray().then(data => {
+      for (const match of data) {
+        for (const matchTeam of match.matchTeams) {
+          if (matchTeam.teamNumber == teamNumber) {
+            matchTeam.teamNumber = undefined
```

Comment:
Didn't we say you would delete this data? What meaning does this connection have if there is no team on the other side?

Reply:
Each match has represantion for all table, and if there is no team in thats table, we set the team number to undifined